pages/features.html

```diff
@@ -6,104 +6,134 @@
 
 {% block body %}
 <div class="content">
-  <H1>{% trans "Everything You Need, However You DJ" %}</H1>
+  <h1>{% trans "Everything You Need, However You DJ" %}</h1>
 
   <div style="clear: both;"></div>
   <div class="halfbox_left">
-    <H2>{% trans "Four Decks, Advanced Controls" %}</H2>
+    <h2>{% trans "Four Decks" %}</h2>
     <p>
-      {% trans "Drop a song onto a deck in Mixxx to kick off your next mix. Each deck features a scratchable, scrolling waveform that marks beats and cue points, along with a whole-song waveform overview for quick seeking." %}
+      {% trans "Drop a song onto a deck in Mixxx to start playing right away. Keep your eye on beats and cue points with a scratchable, scrolling waveform together with a whole-song overview waveform for quick seeking." %}
     </p>
 
-    <p><b>{% trans "Time Stretch and Vinyl Emulation" %}</b><br>
-      {% trans "Speed up and slow down songs without changing their pitch with time stretching, or just change the musical pitch so your songs play in harmony. Our high-quality interpolator can also reproduce classic vinyl sounds like backspins." %}
+    <p><b>{% trans "Pitch and Key Control" %}</b><br>
+      {% trans "Change the tempo of songs without changing their pitch with keylock, or change the pitch apart from tempo so your songs play in harmony. Temporarily nudge the tempo faster or slower for manual beatmatching, with ramping pitchbend smoothly applied like vinyl." %}
     </p>
 
     <p><b>{% trans "Beat Looping" %}</b><br>
-      {% trans "Found a sweet loop or need to extend your mix while you prep the next track? Instantly loop a 4, 8, or 16 beat segment with the press of a button." %}
+      {% trans "Need to extend your mix while you bring in the next track? Loop a 4, 8, or 16 beat segment with ease." %}
```

Comment:
I'll put "instantly" back, but some controllers use an encoder push instead of a button to set a loop.